spyderlib/spyder.py

```diff
@@ -1132,13 +1132,14 @@ def add_xydoc(text, pathlist):
             self.apply_statusbar_settings()
 
             # Third-party plugins
-            for mod in get_spyderplugins_mods(prefix='p_', extension='.py'):
+            for mod in get_spyderplugins_mods():
                 try:
                     plugin = mod.PLUGIN_CLASS(self)
                     self.thirdparty_plugins.append(plugin)
                     plugin.register_plugin()
-                except AttributeError as error:
+                except Exception as error:
                     print("%s: %s" % (mod, str(error)), file=STDERR)
+                    traceback.print_exc(file=STDERR)

```

Comment:
I think it's to have both the plugin name and the full traceback printed. Both are very useful to plugin or core developers to get informations from users even outside debugging phase so I vote to keep both of them.